src/item.cpp

```diff
@@ -10348,20 +10367,13 @@ int item::ammo_remaining( const Character *carrier, bool cable_links ) const
     }
 
     // Cable connections
-    if( cable_links && contents_linked ) {
-        for( const item *cable : contents.cables( true ) ) {
-            if( !cable->link ) {
-                continue;
-            }
-            if( cable->link->t_veh_safe ) {
-                ret += cable->link->t_veh_safe->connected_battery_power_level().first;
-                continue;
-            } else {
-                const optional_vpart_position vp = get_map().veh_at( cable->link->t_abs_pos );
-                if( vp ) {
-                    ret += vp->vehicle().connected_battery_power_level().first;
-                    continue;
-                }
+    if( include_linked && link_length() >= 0 && link->efficiency >= 0.001f ) {
```

Comment:
minor nitpick, but please make the `0.001f` a constexpr somewhere since it's used in several places with no obvious justification.